components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/UpdateAdvancedConfigurations.java

```diff
@@ -79,6 +84,51 @@ private void updateCertificate(Certificate certificate, ServiceProvider serviceP
         }
     }
 
+    private void updateExternalizedConsentPage(ExternalizedConsentPageConfiguration externalizedConsentPageApiModel,
+                                                 LocalAndOutboundAuthenticationConfig config) {
+
+        ExternalizedConsentPageConfig externalConsentManagementConfig = getExternalizedConsentPageConfig(config);
+        if (externalizedConsentPageApiModel != null) {
+            if (isValidConsentPageUrl(externalizedConsentPageApiModel.getConsentPageUrl())) {
+                setIfNotNull(externalizedConsentPageApiModel.getEnabled(),
+                        externalConsentManagementConfig::setEnabled);
+                setIfNotNull(externalizedConsentPageApiModel.getConsentPageUrl(),
+                        externalConsentManagementConfig::setConsentPageUrl);
+            } else {
+                throw buildBadRequestError("Only https consent page urls are allowed ");
```

Comment:
Is the error message format consistent. (There is a space at the end)

Reply:
Resolved by https://github.com/wso2/identity-api-server/pull/434/commits/98196e6ff5d4749f1af21cc83950974389c04625